Configurations/windows-makefile.tmpl

```diff
@@ -362,7 +362,7 @@ libclean:
 	-del /Q /F $(LIBS) libcrypto.* libssl.* ossl_static.pdb
 
 clean: libclean
-	{- join("\n\t", map { "-del /Q /F $_" } @PROGRAMS) -}
+	@{- join("\n\t", map { "-del /Q /F $_" } @PROGRAMS) -}
```

Comment:
With that, you end up having the first `del` hidden.  Bad choice in my opinion

Reply:
Really?  The `del` is on the next line.  Is Window make that different from other versions?

Reply:
Er, did you actually look at the perl?

Reply:
Ah, first `del` is this is included.  I'll close this unrequited.